src/helpers/checkTitleSpelling.tsx

```diff
@@ -0,0 +1,23 @@
+export const checkTitleSpelling = (enteredTitle: string) => {
+  let preparedTitle = '';
+
+  for (let i = 0; i < enteredTitle.length; i += 1) {
+    const charCode = enteredTitle.charCodeAt(i);
+
+    if (charCode === 32
+      || charCode === 1100
+      || charCode === 1108
+      || charCode === 1110
+      || charCode === 1111
+      || (charCode > 47 && charCode < 58)
+      || (charCode > 64 && charCode < 91)
+      || (charCode > 96 && charCode < 123)
+      || (charCode > 1025 && charCode < 1066)
+      || (charCode > 1069 && charCode < 1098)
+      || (charCode > 1101 && charCode < 1105)) {
+      preparedTitle += enteredTitle[i];
+    }
```

Comment:
I'm not sure why are you doing it, but if you want to validate and get rid of some characters you can use regular expressions.
Also, maybe it's not necessary to do it on every change, you can do it when the user submits.
And I think if some invalid characters appear you should notify the user that it's not valid as the user doesn't know why they type and nothing happens